include/boost/geometry/algorithms/detail/within/within_no_turns.hpp

```diff
@@ -142,8 +145,8 @@ template <typename Geometry1,
           typename Geometry2,
           typename Tag1 = typename geometry::tag<Geometry1>::type,
           typename Tag2 = typename geometry::tag<Geometry2>::type,
-          bool IsMulti1 = boost::is_base_of<geometry::multi_tag, Tag1>::value,
-          bool IsMulti2 = boost::is_base_of<geometry::multi_tag, Tag2>::value>
+          bool IsMulti1 = std::is_base_of<geometry::multi_tag, Tag1>::value,
+          bool IsMulti2 = std::is_base_of<geometry::multi_tag, Tag2>::value>
```

Comment:
This could also be replaced with `util::is_multi<Geometry2>::value` from `<boost/geometry/util/type_traits.hpp>`

Reply:
That's right. So this was not the right fix either ;-)
✅ fixed, and on 3 other places as well.